stable/mediawiki/templates/deployment.yaml

```diff
@@ -19,11 +19,26 @@ spec:
         image: "{{ .Values.image }}"
         imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
         env:
+        - name: ALLOW_EMPTY_PASSWORD
+        {{- if .Values.allowEmptyPassword }}
+          value: "yes"
+        {{- else }}
+          value: "no"
+        {{- end }}
+        - name: MARIADB_ROOT_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "mediawiki.mariadb.fullname" . }}
+              key: mariadb-root-password
         - name: MARIADB_HOST
           value: {{ template "mediawiki.mariadb.fullname" . }}
         - name: MARIADB_PORT_NUMBER
           value: "3306"
-        - name: MARIADB_PASSWORD
+        - name: MEDIAWIKI_DATABASE_NAME
+          value: {{ default "" .Values.mariadb.mariadbDatabase | quote }}
+        - name: MEDIAWIKI_DATABASE_USER
+          value: {{ default "" .Values.mariadb.mariadbUser | quote }}
+        - name: WORDPRESS_DATABASE_PASSWORD
```

Comment:
@tompizmor this needs to be `MEDIAWIKI_DATABASE_PASSWORD`  probably :)

Reply:
yes!